source/common/http/async_client_impl.h

```diff
@@ -298,8 +300,8 @@ class AsyncStreamImpl : public AsyncClient::Stream,
     static const NullHedgePolicy hedge_policy_;
     static const NullRateLimitPolicy rate_limit_policy_;
     static const Router::InternalRedirectPolicyImpl internal_redirect_policy_;
-    static const Router::PathMatchPolicyImpl path_match_policy_;
-    static const Router::PathRewritePolicyImpl path_rewrite_policy_;
+    static const Router::PathMatcherSharedPtr extension_path_matcher_;
```

Comment:
nit: I'm not sure how much value "extension_" provides. I'd just do `path_matcher_` and `path_rewriter_` (especially since that matches the getters)

Reply:
removed

Reply:
It doesn't seem to be changed according to the current contents of the PR in github. Do you need to push a new patch set?

Reply:
I see a change in source/common/router/config_impl.h, but not in this file.